src/client/components/login-form.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      result: ""
+    };
+  }
+  clickHandlerCancel = event => {
+    this.setState({
+      email: "",
+      password: ""
+    });
+  };
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = event => {
+    event.preventDefault();
+
+    try {
+      fetch(`${process.env.API_PATH}/auth/login/`, {
```

Comment:
We need to use webpack to print the env vars on compile. Like in this article: https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5

Tried getting it to work today, but was not succesful. Will give it another shot tomorrow.